index.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://www.1001fonts.com/cocogoose-pro-font.html">
+    <link rel="stylesheet" href="css1.css">
+    <title>Capstone</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <div class="part1nav none">
+                <img src="images/part1nav.PNG" alt="">
+            </div>
+            <a class="none logo" href ="#">
+                <img src="images/logo.PNG" alt="">
+            </a>
+        <a class="none2 humberger" href="#">___</a>
+
+         <ul class="ulnav">
+            <li class="linav"><a class="a" href="About.html">About</a> <hr class=" hrnav none2"> </li>
+            <li class="linav"><a class="a b" href="#program">Program</a><hr class=" hrnav none2"> </li>
+            <li class="linav"><a class="a b2" href="#speakers">Speakers</a><hr class=" hrnav none2"> </li>
+            <li class="linav"><a class="a b3" href="#Sponsors">Sponsors</a><hr class=" hrnav none2"> </li>
+        </ul>
+        </nav>
+        <div class="header-description">
+        <h1 class="h1-header">Hello! sharing world</h1>
+        <h2 class="h2-header">CREATIVE COMMONS GLOBAL SUMMIT 2015</h2>
+        <p class="p-header">A joyful celebration believing in the value of oppenness and sharing,
+         creating a positive change with people from over 80 countries is taking place in Octobar, in Korea</p>
+         <h3 class="h3-header">2015.10.15(THU) ~ 16(FRI)</h3>
+         <H4 class="h4-header">@National Museum od Korea,Art center Nabi</H4>
+        </div>
+    </header>
```

Comment:
- [ ] Kindly make all the changes asked in the video attached. 
[Review Loom Video](https://www.loom.com/share/35eb0c18a5d64b9fbf5b2a022f3b328f)
- [ ] Basically, you have to make changes in colour, font size, font weight, alignment, spacing etc as suggested in the video.
- [ ] You have to make changes to the mobile version of the speaker section too as suggested in the video.
- [ ] Make sure to add a home option for the mobile menu as suggested in the video.
- [ ] Kindly remove the extra white spacing as suggested at the end of the video.
- [ ] Kindly make sure that all the images in the speaker section have background checks image at the left top side as suggested in the video.
- [ ] Make sure to follow the same background colour as given in the design as suggested in the video.